.github/workflows/ci.yml

```diff
@@ -14,9 +14,9 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.16
+          go-version: 1.17
       - name: Test
-        run: go test -v -race -coverpkg=./... -covermode=atomic -coverprofile=coverage.out ./...
```

Comment:
Should we add a comment about why `-race` is disabled or maybe we can run the race detector in a job that is allowed to fail?